hotline/pages/forms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms
+from django.contrib.flatpages.forms import FlatpageForm
+from django.contrib.sites.models import Site
+from django.utils.translation import ugettext
+
+reserved = ['edit', 'create', 'list', 'delete']
```

Comment:
Could you use something like https://docs.djangoproject.com/en/1.8/ref/urlresolvers/#resolve so you don't have to hard code these special names? For example, call resolve on the URL in the clean method, and ensure it raises a `Resolver404`?